requirements/aarch64.txt

```diff
@@ -5,3 +5,7 @@ aiomysql==0.0.22
 aioboto3==9.0.0
 motor==2.5.1
 smbprotocol==1.9.0
+SQLAlchemy==1.4.44
```

Comment:
since it's the exact same list than x86_64, let's move them to the framework.txt one

we also need for each new lib to check the license, if you can list them in the issue, so we check that we don't introduce a lib that is not in the list of allowed licenses for Elastic thx

Reply:
Ok move all libs to framework.txt